Pipfile

```diff
@@ -14,4 +14,4 @@ pydocstyle = "*"
 pylint = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
Shouldn't `Pipfile.lock` also change in the same commit?

Reply:
Good catch, I'll update.